src/content/en/updates/2018/03/present-web-pages-to-secondary-attached-displays.md

```diff
@@ -0,0 +1,268 @@
+new window, which the user has to then manually drag to the secondary display
+and maximize to fullscreen), it is cumbersome and provides an inconsistent
+experience between local and remote presentation.
+
+Key Point: This change is about enabling secondary, attached displays to be
+used as endpoints for presentations, in the same way as remote endpoints.
+
+## Present a page
+
+Let me walk you through how to use the [Presentation API] to present a web page
+on your secondary attached display. The end result is available at
+[https://googlechrome.github.io/samples/presentation-api/].
+
+First, we’ll create a new `PresentationRequest` object that will contain the
+URL we want to present on the secondary attached display.
+
+    const presentationRequest = new PresentationRequest('receiver.html');
+
+In this article, I won’t cover use cases where the parameter passed to
+`PresentationRequest` can be an array like `['cast://foo’, 'apple://foo',
+'https://example.com']` as this is not relevant there.
+
+We can now monitor presentation display availability and for instance toggle a
+"Present" button visibility based on presentation displays availability. Note
+that we can also decide to always show this button. 
+
+Caution: the browser may use more energy while the `availability` object is alive
+and actively listening for presentation display availability changes. Please
+use it with caution in order to save energy on mobile.
+
+    presentationRequest.getAvailability()
+    .then(availability => {
+      console.log('Available presentation displays: ' + availability.value);
+      availability.addEventListener('change', function() {
+        console.log('> Available presentation displays: ' + availability.value);
+      });
+    })
+    .catch(error => {
+      console.log('Presentation availability not supported, ' + error.name + ': ' +
+          error.message);
+    });
+
+Showing a presentation display prompt requires a user gesture such as a click
+on a button. So let’s call `presentationRequest.start()` on a button click for
+instance and wait for the promise to resolve once user has selected a
+presentation display (.e.g. a secondary attached display in our use case).
+
+    function onPresentButtonClick() {
+      presentationRequest.start()
+      .then(connection => {
+        console.log('Connected to ' + connection.url + ', id: ' + connection.id);
+      })
+      .catch(error => {
+        console.log(error);
+      });
+    }
+
+List presented to the user may also include remote endpoints such as Chromecast
+devices if you’re connected to a network advertising some.
+
+<figure>
+  <img src="/web/updates/images/2018/03/present-web-pages-to-secondary-attached-displays/picker.png"
+    alt="Presentation Display Picker"/>
+</figure>
+
+When promise resolves, the web page at the `PresentationRequest` object URL is
+presented to the attached display user has previously picked. And voila!
```

Comment:
'attached display user has previously picked' > 'chosen display'

(Side note. Some writers might actually write 'Et voila!' here. As a native English speaker, it would sound pretentious if I did it. It would not be pretentious for you.)

Reply:
Done.

Thanks for the side note, I'll use "Et voila!" sporadically ;)